docs/operator/operations.md

```diff
@@ -0,0 +1,100 @@
+# Lifecycle Manager - Operations
+
+## Metrics
+
+Lifecycle Manager metrics are exposed over port `8080` through the metrics endpoint `/metrics`.
+
+The following metrics are exposed:
+
+| Metric Name                              | Metric Type    | Description                                                                                  |
+|------------------------------------------|----------------|----------------------------------------------------------------------------------------------|
+| `lifecycle_mgr_requeue_reason_total`     | Counter Vector | Indicates the requeue reason of a reconciler.                                                |
+| `lifecycle_mgr_kyma_state`               | Gauge Vector   | Indicates the state of a Kyma CR.                                                            |
```

Comment:
Where can we find the possible values for the states, and their actual meaning for an operator?